libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/InsightDrillConfigPanel/useInsightDrillConfigPanel.ts

```diff
@@ -256,15 +258,22 @@ function buildBlacklistHierarchies(item: IDrillDownAttributeHierarchyConfig): ID
             {
                 type: "dateHierarchyReference",
                 dateHierarchyTemplate: item.attributeHierarchyRef,
-                dateDatasetAttribute: idRef(attributeDescriptor.attributeHeader.identifier, "displayForm"),
+                dateDatasetAttribute: idRef(
+                    attributeDescriptor.attributeHeader.formOf.identifier,
+                    // Use "displayForm" because the backend hasn't supported attribute as reference
+                    // when we save the widget
+                    "displayForm",
+                ),
             },
         ];
     }
     return [
         {
             type: "attributeHierarchyReference",
             attributeHierarchy: item.attributeHierarchyRef,
-            label: idRef(attributeDescriptor.attributeHeader.identifier, "displayForm"),
+            // Use "displayForm" because the backend hasn't supported attribute as reference
+            // when we save the widget
+            label: idRef(attributeDescriptor.attributeHeader.formOf.identifier, "displayForm"),
```

Comment:
Please, ask in #team-core how complicated it would be to have this as: 

````
type: "attributeHierarchyReference",
attributeHierarchy: item.attributeHierarchyRef,
attribute: idRef(<attribute-id>, "attribute")
 ````

Saving attribute id in label is not good idea imho.